src/styles/main.scss

```
@@ -1,3 +1,54 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
 body {
   margin: 0;
 }
+
+.blocks {
+  display: grid;
+  grid-template: repeat(6, minmax(300px, 1fr)) /
+    repeat(auto-fill, minmax(300px, 1fr));
+
+  @media (min-width: 1200px) {
+    & {
+      grid-template-columns: repeat(3, minmax(300px, 1fr));
+    }
```

```suggestion
    grid-template-columns: repeat(3, minmax(300px, 1fr));
```
Зачем тебе эта обертка?